Automattic/vip-go-mu-plugins
# Add Jetpack 9.4 subtree

## Description

Add a new Jetpack subtree version 9.4 for the 9.4-beta version released today.

## Changelog Description

### Added Jetpack 9.4-beta

We have added a new Jetpack subtree for minor 9.4, initially containing version 9.4-beta. If you want to try this beta, you can follow the [testing guidelines](https://github.com/Automattic/jetpack/blob/jetpack/branch-9.4/to-test.md) from Jetpack.

As usual, you need to pin your site to use Jetpack 9.4 by adding this line to `vip-config.php`

```
define( 'VIP_JETPACK_PINNED_VERSION', '9.4' );
```

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Check out PR.
1. Pin the testing site to Jetpack 9.4: `define( 'VIP_JETPACK_PINNED_VERSION', '9.4' );`
1. Ensure version 9.4-beta is active on the Jetpack dashboard, and that everything keeps working.
